finalPrep/modZeroHero.js

```diff
@@ -55,30 +96,72 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name: "Godzilla",
+  smell: "Stinky",
+  weight: 500,
+  citiesDestroyed: ["Cancun", "Miami"],
+  luckyNumbers: [7, 21, 3],
+  address: {
+    number: 12,
+    street: "16th Street",
+    state: "Kansas",
+    zip: 80203
+  }
+}
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+class SuperHero {
+  constructor(){
+    this.archEnemies = "The Syntax Error";
+    this.powerLevelr = "floppy";
+    this.energyLevel = 50;
+  }
+  changeName(){
+    this.name = "Superman";
+  } changeSuperpower(){
+    this.superpower = "invicible";
+  } addAge(){
+      this.age = "55";
+  }
+}
+
+var batman = new SuperHero();
 
 // - Create the following class methods
```

Comment:
The `sayName`, `maximizeEnergy`, and `gainPower` methods should exist on your `SuperHero` class above.  You can delete the `favoriteHero` class.